src/helm/benchmark/presentation/summarize.py

```diff
@@ -158,6 +151,38 @@ def get_method_display_name(model_display_name: Optional[str], info: Dict[str, A
     return (model_display_name or "???") + (f" [{dict_to_str(info)}]" if len(info) > 0 else "")
 
 
+def compute_row_win_rates(table: Table) -> List[Optional[float]]:
```

Comment:
probably should average in the name. 

One could imagine a slightly more involved abstract where:
1. You have a function that computes for each row its win rate/rank for each columns. So if the original table was `m x n`, this is still an `m x n` object.
2. Given this `m x n` matrix, where each cell is the win rate of the given model (row) for the given scenario/metric (column), you then compute the column-wise average to get the `row_average_win_rate`. And the point is this is separate from 1 in the sense you could do stuff other than the average here (e.g. the max or min or some weighted average which implies winning in some columns is more important than others). 

For now I don't think we need this complexity/we could just have this as-is, but could even just mention the possibility for other things in the documentation.

Reply:
(added average to the name)

Good point. In some sense the function is already factorized like this, so it should be an easy extension if we want to do something else. Will leave as-is for now to keep things more self-contained but will add a note.

Reply:
cool, makes sense to me!